exercises/02-publish-subscribe/README.md

```diff
@@ -407,4 +407,10 @@ This is a powerful feature which could be of huge value to operations activities
 We now have a dashboard that can inform us when an endpoint is  or messages failed to be processed. A few things to consider:
 
 - No-one wants to watch the dashboard all day. Fortunately, ServiceControl also uses pub/sub to notify subscribers of events. You can build an endpoint that subscribes to ServiceControl events and informs you of downtime or failed email, SMS or other means. Read more about [using ServiceControl events](https://docs.particular.net/servicecontrol/contracts).
-- You might notice several endpoints with the same name. Endpoints send heartbeats with a unique host identifier, made up of their endpoint name and a hash of the folder the endpoint is installed in. Our exercises all have the same endpoint name, but different folders. Another example is when you deploy endpoints using [Octopus](https://octopus.com/). This will deploy every version in its own folder, with the result that every version will spawn a new monitored endpoint in ServicePulse. You can solve this by [overriding the host identifier](https://docs.particular.net/nservicebus/hosting/override-hostid.
+- You might notice several endpoints with the same name. Endpoints send heartbeats with a unique host identifier, made up of their endpoint name and a hash of the folder the endpoint is installed in. Our exercises all have the same endpoint name, but different folders. Another example is when you deploy endpoints using [Octopus](https://octopus.com/). This will deploy every version in its own folder, with the result that every version will spawn a new monitored endpoint in ServicePulse. You can solve this by [overriding the host identifier](https://docs.particular.net/nservicebus/hosting/override-hostid).
+
+## Conclusion
+
+Hopefully this exercise has demonstrated how you can use the publish/subscribe messaging pattern to decouple services.
```

Comment:
Nitpick: "publish/subscribe" should be "Publish-Subscribe" to be consistent with the rest of the document.

(There is one other existing inconsistency, and also we should lower case the term throughout, but that is something for another PR.)